### DISCUSSION TITLE
ZBMini Switchmode?

### DISCUSSION THREAD

**Discussion by brookb52** [arendst/Tasmota#12869] [Q&A] [answered]
I have been using Tasmotized Sonoff Minis and MiniR2s for a while now and decided to switch to Zigbee to give me back some much needed space on my IP network. I think I heard my router give a sigh of relief.
I use the SwitchMode feature of Tasmota where I can change the action of the s1 and s2 switch terminals and was wondering if it was possible to change the SwitchMode for the ZBMini.

**Comment by digiblur** [comment, marked as answer]
Without rewriting your own firmware on the ZBMini it isn't possible as I didn't see the option in the stock firmware pairing methods.  

How many devices did you have?  There's a lot of room on internal private IPs.  I'm at probably 100 ESP based devices with lots of room to go even on the spectrum side.

**Reply by brookb52** [reply]
About 70 devices but going by your setup I see that shouldn't be a problem.

I thought rewriting the firmware might be the case.
Thanks for the reply.
